wrangler-api/src/main/java/io/cdap/wrangler/api/ErrorRowException.java

```diff
@@ -42,7 +39,7 @@ public ErrorRowException(String message, int code) {
   }
 
   public ErrorRowException(String directiveName, String errorMessage, int code) {
-    this(String.format("Error encountered while executing '%s' : %s", directiveName, errorMessage), code);
+    this(String.format("%s (ecode: %d, directive: %s)", errorMessage, code, directiveName), code);
```

Comment:
how about if we first have directive name, error message and then error code?

Reply:
Directive is secondary information, I would like to know what the error is first and then find out where it's coming from. That's why I changed the order.